### PR TITLE
Error Tooltip

### PR DESCRIPTION
**Pull request by DarksightKellar**
Displays a tooltip on error icon hover in Review Sarco screen (should also work wherever `SummaryErrorIcon` is used).

### CODE REVIEW

**Review by knoll3 — APPROVED** [review]

File: src/features/embalm/stepContent/components/SummaryErrorIcon.tsx [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
-    </Flex>
+      <Flex
+        h="1.2rem"
+        w="1.2rem"
+        borderRadius={100}
+        background="errorAlt"
+        alignItems="center"
+        justifyContent="center"
+      >
+        <Text color="red">!</Text>
```
Comment (mobiustripper42) [inline src/features/embalm/stepContent/components/SummaryErrorIcon.tsx]: very minor point, but with the `!` being `<Text>`. We get the I bar selector, and in this screenshot you can see it's possible to select the `!`. Perhaps something like ...

`      <Flex
        h="1.2rem"
        w="1.2rem"
        borderRadius={100}
        background="errorAlt"
        alignItems="center"
        justifyContent="center"
        color="red"
      >
        <Button variant="unstyled">!</Button>
      </Flex>`

**Review by mobiustripper42 — APPROVED** [review]
looks good

**Review by mobiustripper42 — APPROVED** [review]

**Review by mobiustripper42 — APPROVED** [review]
looks good

File: src/features/embalm/stepContent/components/SummaryErrorIcon.tsx (hunk 1 above)
Comment (DarksightKellar) [reply src/features/embalm/stepContent/components/SummaryErrorIcon.tsx]: This is cool and muuch better. Thanks!

**Pull request merged**